cves/CVE-2013-2251.yml

```diff
@@ -103,9 +103,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: false
+  code: false
+  fix: false
```

Comment:
Why do you say there were no tests? From the fix commit, I noticed that changes had been made to a test file (core/src/test/java/org/apache/struts2/dispatcher/mapper/DefaultActionMapperTest.java). It looks like the author added a test for unsafe redirects and made a few more changes to other tests. It seems like the unit tests weren't testing the correct things previously to catch this vulnerability.

Reply:
I added a description and changed code to true. You are right, there are tests, just not for this vulnerability. They also weren't added after the patch. I believe the tests apply to `https://cwiki.apache.org/confluence/display/WW/S2-017` instead.